README.md

```diff
@@ -877,14 +878,12 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
   * [tastebin](https://github.com/andreineculau/tastebin) - Updated version of hastebin plus additional features. `Apache` `Nodejs`
 
 
-## Personal Dashboards
+## Personal Dashboards 
+See [here](https://github.com/n1trux/awesome-sysadmin#metric--metric-collection)
```

Comment:
Please just use 

`See https://github.com/n1trux/awesome-sysadmin#metric--metric-collection`

for consistency with other similar links.